assets/dev/js/editor/elements/views/widget.js

```diff
@@ -234,6 +234,35 @@ WidgetView = BaseElementView.extend( {
 			}, 200 );
 			// Is element empty?
 		} );
+
+		const $dropArea = self.$el.find( '.elementor-drop-area' );
+
+		if ( $dropArea.length ) {
+			$dropArea.html5Droppable( {
+				axis: [ 'vertical' ],
+				isDroppingAllowed: true,
+				currentElementClass: 'elementor-html5dnd-current-element',
+				placeholderClass: 'elementor-sortable-placeholder elementor-widget-placeholder',
+				hasDraggingOnChildClass: 'elementor-dragging-on-child',
+				onDropping: ( side, event ) => {
+					event.stopPropagation();
+
+					// TODO: Check with @kobizz if it's necessary.
+					// Triggering drag end manually, since it won't fired above iframe.
+					elementor.getPreviewView().onPanelElementDragEnd();
+
+					const elementView = elementor.channels.panelElements.request( 'element:selected' );
+					const widgetType = elementView.model.get( 'widgetType' );
+					const targetId = this.getContainer().id;
+					const tabId = event.currentTarget.dataset.tab;
+
+					console.log( `Dropped: ${ widgetType }` );
+					console.log( `Target: ${ targetId }` );
+					console.log( `Inner Element: ${ tabId }` );
```

Comment:
We need the repeater item id, in order to make it generic, try to add the item id as a data attribute on the `elementor-drop-area` element.